### ISSUE TITLE
release 4.0.2 broke importing and running commands programmatically. 

### ISSUE THREAD

**Issue by dopry** [auth0/auth0-deploy-cli#213]
### Description

When importing deploy from a scipt the following is displayed error 
```
Auth0 Deploy CLI

Commands:
  auth0-import.js import  Deploy Configuration  [aliases: deploy]
  auth0-import.js export  Export Auth0 Tenant Configuration  [aliases: dump]

Options:
  --help           Show help  [boolean]
  --version        Show version number  [boolean]
  --debug, -d      Dump extra debug information.  [string] [default: false]
  --proxy_url, -p  A url for proxying requests, only set this if you are behind a proxy.  [string]

Examples:
  auth0-import.js export -c config.json -f yaml -o path/to/export       Dump Auth0 config to folder in YAML format
  auth0-import.js export -c config.json -f directory -o path/to/export  Dump Auth0 config to folder in directory format
  auth0-import.js import -c config.json -i tenant.yaml                  Deploy Auth0 via YAML
  auth0-import.js import -c config.json -i path/to/files                Deploy Auth0 via Path
  auth0-import.js dump -c config.json -f yaml -o path/to/export         Dump Auth0 config to folder in YAML format
  auth0-import.js dump -c config.json -f directory -o path/to/export    Dump Auth0 config to folder in directory format
  auth0-import.js deploy -c config.json -i tenant.yaml                  Deploy Auth0 via YAML
  auth0-import.js deploy -c config.json -i path/to/files                Deploy Auth0 via Path

See README (https://github.com/auth0/auth0-deploy-cli) for more in-depth information on configuration and setup.

A command is required
```

I expect the deploy method to be imported without executing the validator. 

### Reproduction

`echo  "require('auth0-deploy-cli')" | node`

### Environment
v4.0.2

**Comment by brianmcd** [comment]
I'm experiencing this, too.  Seems likely that it would be related to https://github.com/auth0/auth0-deploy-cli/pull/192?

**Comment by dmi3y** [comment]
Confirmed, it is not working from module default export. My workaround:
> import dump from "auth0-deploy-cli/lib/commands/export"

**Comment by shawnmclean** [comment]
This has been fixed in 4.0.3 😄: https://github.com/auth0/auth0-deploy-cli/releases/tag/v4.0.3
